package.json

```diff
@@ -25,19 +25,64 @@
   "peerDependencies": {
     "@angular/common": ">=4.3.6",
     "@angular/core": ">=4.3.6",
-    "@skyux/grids": "^3.0.0",
-    "@skyux/i18n": "^3.5.0",
-    "@skyux/indicators": "^3.0.0",
-    "@skyux/list-builder": "^3.2.1",
-    "@skyux/list-builder-view-checklist": "^3.0.0",
+    "@skyux/grids": "^3.2.0",
+    "@skyux/i18n": "^3.7.0",
+    "@skyux/indicators": "^3.1.0",
+    "@skyux/list-builder": "^3.5.0",
+    "@skyux/list-builder-view-checklist": "^3.1.0",
```

Comment:
Do we need to update the peers?

Reply:
Maybe not in this PR, but I noticed a serious issue - if you install the latest `view-grids`, but only have grids `3.0` installed, you will have an issue when trying to use the `settingsKey` input --> it didn't exist until version `3.1`. Figured I could kill two birds with one stone here. Think that's bad form?

Reply:
Not at all. I'm okay with updating any of the peers as long as it's a requirement.